app/views/users/show.html.erb

```diff
@@ -1 +1,23 @@
-<h1>Here is all user history information</h1>
\ No newline at end of file
+<h1>Here is all <%= @user.name %> history information</h1>
+
+<div class="users-container">
+  <img class="user-photo" src="<%= @user.photo %>"><br>
+  <h2><%= @user.name %></h2><br>
+  <p class="text-usercard-index">Number of Posts: <%= @user.posts_counter %></p><br><br>
+</div>
+<div class="bio-container">
+  <p class="text-bio">
+   Bio:<br><br>
+   <%= @user.bio %>
+  </p>
+</div>
+<% @user.posts.each_with_index do |post, index| %>
+<div class="posts-container">
+  <div class="posts-title-container">
```

Comment:
* Kindly add missing functionality from wireframes provided in the project requirement as currently, i can only see the index view for all users but views for each user which should take you to their show page [As shown in this image ](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=AMQQWXSOCNGGS6YXF3P7WDDBYO76G) NOTE ✍️  they are four images which serve as a wireframe the functionalities shown there are required for this project [Refer back to the project requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Project%20requirements,everything%20functions%20first.)

Reply:
> * Kindly add missing functionality from wireframes provided in the project requirement as currently, i can only see the index view for all users but views for each user which should take you to their show page [As shown in this image ](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_page.png?token=AMQQWXSOCNGGS6YXF3P7WDDBYO76G) NOTE ✍️  they are four images which serve as a wireframe the functionalities shown there are required for this project [Refer back to the project requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Views_project.md#:~:text=Project%20requirements,everything%20functions%20first.)

Hi @Davidosky007 Thanks for your review 👍, please kindly can you check again? because the change you are requiring is already applied, kindly, can you check the followings paths?

users index (1st)
![image](https://user-images.githubusercontent.com/82678907/146287346-85f0dec3-b597-4036-9fc2-fc28c989c5e4.png)

 user show (2nd)
![image](https://user-images.githubusercontent.com/82678907/146287464-c423c966-53d4-4784-bc52-5f7b2dbe3a8c.png)


Posts index(3rd)
![image](https://user-images.githubusercontent.com/82678907/146287513-bef56af3-c408-401e-aba3-1d5a2bbdfe74.png)


 and post-show (4rth)
![image](https://user-images.githubusercontent.com/82678907/146287633-978f5538-5ae3-470a-8c36-12360873c945.png)

It's maybe a configuration error?, please see the readme file to install the database with default information,

I really appreciate it, if you can review it again. Thank you so much @Davidosky007 , I will be waiting your answer 🤓